### PR TITLE
feat/analytics 1.9

### LINKED ISSUE #3717: Data analytics 1.9
- [X] track events only from production builds to `stable.log`, all others to `develop.log`
- [ ] session-end is not sent on desktop app close: _Probably no problems after implementing Beacon API_
- [X] update docs
- [X] track suite-ready on initial run
- [X] analyse new events [Notion](https://www.notion.so/satoshilabs/Analytics-overview-3b49d66235e044928b79954d3a4ac8ff)
- [X] add new events, extend and fix existing ones 
  - the rest can be done in 1.10 analytics

### PR DESCRIPTION
**Pull request by tomasklim**
closes #3717 

- updated docs
- `suite-ready` event tracked on initial run
- added new events and extensions of old ones
- fixed `transaction-created` event which was not sent at all
- only events from codesign branch build are tracked to `stable.log`, other ones to `develop.log`

[Notion info](https://www.notion.so/satoshilabs/Analytics-overview-3b49d66235e044928b79954d3a4ac8ff)

### CODE REVIEW

File: packages/suite/src/actions/suite/analyticsActions.ts [hunk 1]
```diff
@@ -391,19 +391,23 @@ export const report = (data: AnalyticsEvent, force = false) => (
     if (initialRun) {
         return;
     }
-    // the only case we want to override users 'do not log' choice is when we
-    // want to log that user did not give consent to logging.
+
+    /**
+     * The only case we want to override users 'do not log' choice is
+     * when we want to log that user did not give consent to logging.
+     */
```
Comment (goodhoko) [inline packages/suite/src/actions/suite/analyticsActions.ts]: I wouldn't use doc comment here. This comments relates to a imperative piece of code and is only relevant here. Doc comments are used to provide additional information on symbols (functions, variables, constants...) that can be referenced eslewhere. I'd keep the `//` format here.

File: packages/suite/src/utils/suite/sentry.ts [hunk 2]
```diff
@@ -5,3 +5,10 @@ export const setSentryUser = (instanceId: string) => {
         scope.setUser({ id: instanceId });
     });
 };
+
+// analytics is disabled during runtime
```
Comment (goodhoko) [inline packages/suite/src/utils/suite/sentry.ts]: This comment confuses me. Is it saying that this function is called when analytics are disposed? If so, I'd rather
remove it - it's not very maintainable to track where a function is used in a comment above its declaration. You can't be sure that next time someone will use this function elsewhere they'll update to comment accordingly. Let's document what the function does (The mere name is enough here, I think.) and leave tracking the usages to IDEs. .)

```suggestion
```

File: packages/suite/src/actions/suite/analyticsActions.ts [hunk 3]
```diff
@@ -33,14 +33,8 @@ export type AnalyticsAction =
...
-/**
-simple semver for data-analytics part.
-<breaking-change>.<analytics-extended>
-
-Don't forget to update docs with changelog!
-*/
-
-export const version = '1.8';
+//  Don't forget to update docs with changelog!
+export const version = '1.9'; // <breaking-change>.<analytics-extended>
```
Comment (goodhoko) [inline packages/suite/src/actions/suite/analyticsActions.ts]: I tend to always write comments on a separate line rather than to append them to the code. It imo reads nicer and doesn't risk being out of viewport in GUI.

```suggestion
 // <breaking-change>.<analytics-extended>
 export const version = '1.9';
```

File: packages/suite/src/components/suite/modals/ReviewTransaction/components/OutputList.tsx [hunk 4]
```diff
@@ -109,6 +109,33 @@ const OutputList = ({
         return undefined;
     };
 
+    const analytics = useAnalytics();
+    const reportTransactionCreatedToAnalytics = (
```
Comment (goodhoko) [inline packages/suite/src/components/suite/modals/ReviewTransaction/components/OutputList.tsx]: nit pick, optional:

```suggestion
    const reportTransactionCreated = (
```

RBY

File: docs/misc/analytics.md [hunk 5]
```diff
@@ -1,24 +1,31 @@
...
+- device id
+- public keys
+- transaction id
+- ... any other fingerprinting
 
-For general overview see [documentation in Notion](https://www.notion.so/e19845789ccb47a0baf36d7f8463f196?v=2183c8228ebb4df38f358d1b2744e1b8).
+## Tracking process
 
-## Anonymity
+Data about interactions are transferred in GET HTTP requests encoded in URI.
```
Comment (goodhoko) [inline docs/misc/analytics.md:16]:
```suggestion
 Data about interactions are transferred in GET HTTP requests encoded in a query string.

**Review by goodhoko — APPROVED** [review]
Very nice. Approving with some marginal remarks.

File: packages/suite/src/utils/suite/device.ts [hunk 6]
```diff
@@ -116,7 +116,7 @@ export const getVersion = (device: TrezorDevice) => {
     return features && features.major_version > 1 ? 'T' : 'One';
 };
 
-export const getFwVersion = (device: AcquiredDevice) => {
+export const getFwVersion = (device: KnownDevice) => {
```
Comment (szymonlesisz) [inline packages/suite/src/utils/suite/device.ts:119]: why this change?
`AcquiredDevice` is extended `KnownDevice`

Comment (tomasklim) [reply packages/suite/src/utils/suite/device.ts:119]: `KnownDevice` always have `features`, so it would be better to have here a more specific type.

I wanted to use it with `prevDevice` which is in fact `Device` (`KnownDevice | UnknownDevice`) from `DEVICE.DISCONNECT` event. I check, that it has `features` so the possibility to be `UnknownDevice` is not valid anymore.

https://github.com/trezor/trezor-suite/pull/3818/files/6984f5fe00bab95510e3a04c1225628a87a16c2a#diff-545d00698b46c7b1a497151825b00ef48aea064f955cf81b2526e8451dfc9458R57

**Comment by tomasklim** [comment]
The last force-push was done to rebase the branch from `feat/3790-fw-btc-only-update` to `develop`

File: packages/suite/src/utils/suite/device.ts (hunk 6 above)
Comment (tomasklim) [reply packages/suite/src/utils/suite/device.ts:119]: `AcquiredDevice` > `KnownDevice`, but `KnownDevice` is enough for this usecase

**Comment by tsusanka** [comment]
The failing job should be fixed in develop soon (#3820). I think let's merge after we make sure #3820 fixes the issue.

**Review by szymonlesisz — APPROVED** [review]
